src/test/java/com/automation/loginTest.java

```diff
@@ -0,0 +1,9 @@
+package com.automation;
+
+public class loginTest {
```

Comment:
@ilianafantano - Can you please make class name Capital?

Reply:
no problem boss